### ISSUE TITLE
Build Fails

### ISSUE THREAD

**Issue by crombiecrunch** [EmberCoin/Ember#11]
`txdb-leveldb.h:15:24:` fatal error: leveldb/db.h: No such file or directory
compilation terminated.
makefile.unix:192: recipe for target 'obj/checkpoints.o' failed
make: *** [obj/checkpoints.o] Error `1`

**Comment by aaashes** [comment]
Came across this when first compiling. You'll need to change the mode bits and then attempt another make:
`$ chmod 755 leveldb/* -v; make -f makefile.unix -j2`

**Comment by crombiecrunch** [comment]
thats the thing at least in this fork that directory doesnt exist. Should i be using the original github?

**Comment by aaashes** [comment]
I cloned the original repo and had no issues. No past commits in this fork have been made to the unix makefile so that shouldn't be the issue. Are you building just the daemon or the qt? Going to clone this repo, confirm my end to insure I haven't missed anything else. For time being you can clone original or my repo (https://github.com/aaashes/Ember) with the added bootstrap til this is resolved.

**Comment by aaashes** [comment]
So leveldb, obj{,-test} and qt dirs are missing from all branches in this fork. @EmberCoin why was no commit made to note these removals?

**Comment by crombiecrunch** [comment]
Ive built off of your repo @aaashes and syncing now. thanks for the help, Ill leave this open until @EmberCoin replies to it

**Comment by EmberCoin** [comment]
I released a daemon build off of windows infrastructure. When I moved on to do Qt I began to see what a mistake this was. You'll see the msvs solutions under ./build/msvs{-qt}/ -- a foolhardy task if I ever do that again.

The old build files and methodology remains however and that's what you're seeing now. Once I'm fully building everything "the keep everything the same" way, with back-ported development from the dev branch, then I will note dependency removals if they have happened and everything still works. MinGW is okay. Once we have a solid base I'll be able to remove some of the build process nonsense and we'll get this show on the road.
